checkov/secrets/plugins/custom_regex_detector.py

```diff
@@ -43,26 +50,84 @@ def analyze_line(
     ) -> Set[PotentialSecret]:
         """This examines a line and finds all possible secret values in it."""
         output: Set[PotentialSecret] = set()
-        for match, regex in self.analyze_string(line, **kwargs):
+
+        self._find_potential_secret(
+            filename=filename,
+            string_to_analyze=line,
+            output=output,
+            line_number=line_number,
+            context=raw_context,
+            is_multiline=False,
+            **kwargs
+        )
+
+        if not self._did_analyzed_file.get(filename):
+            self._did_analyzed_file[filename] = True
+            file_content = None
+            try:
+                with open(filename, 'r') as f:
+                    file_content = f.read()
+            except Exception:
+                logging.error(
```

Comment:
Errors sometimes cause people to think checkov stopped working. Warning, however, tells there was a specific issue here that doesn't affect the entire run. Let's use `warning`
```suggestion
                logging.warning(
```